examples/torchtune/llama3_2/alpaca-trainjob-yaml.ipynb

```diff
@@ -248,7 +276,7 @@
     "from kubeflow.trainer.constants import constants\n",
     "\n",
     "log_dict = client.get_job_logs(job_name, follow=False, step=constants.DATASET_INITIALIZER)\n",
-    "print(log_dict[constants.DATASET_INITIALIZER])"
+    "print(log_dict.get(constants.DATASET_INITIALIZER))"
```

Comment:
@jaiakash Please update the `get_job_logs()` API based on this SDK changes: https://github.com/kubeflow/sdk/pull/83